app/views/pages/GUIDE.markdown

```diff
@@ -35,7 +35,7 @@ Please add yourself to this list (link is optional) if you've contributed to thi
 
 *Updated Summer 2019*
 
-| Course Code |Course Name| Fall | Winter | Summer/Intersession |
+| Course Code | Course Name | Fall | Winter | Summer/Intersession |
```

Comment:
I see why this isn't working. I had to do some hard coding in the markdown renderer to give the table colors/whatnot. This method won't work, I'll have to modify the renderer in a separate PR, created an issue for it #73. You can remove this change.